packages/utils/src/carequality/cq-monthly-stats/xca-dq-stats.ts

```diff
@@ -0,0 +1,47 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { OutboundDocumentQueryResp } from "@metriport/ihe-gateway-sdk";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import {
+  GWWithStats,
+  ImplementerWithGwStats,
+  associateGwToImplementer,
+  queryResultsTable,
+  getDurationsPerGW,
+  RequestParams,
+  QUERY_RESULTS_TABLE_NAMES,
+} from "./shared";
+
+dayjs.extend(duration);
+
+export async function getXcaDqStatsForDay({
+  cqDirectory,
+  endOfPreviousMonth,
+  dayIndex,
+}: RequestParams): Promise<ImplementerWithGwStats[]> {
+  const xcaDQGWStats: GWWithStats[] = await aggregateXcaDqGwStats(endOfPreviousMonth, dayIndex);
+
+  const xcaDqStats: ImplementerWithGwStats[] = await associateGwToImplementer(
```

Comment:
We don't need to `await` here - and all places that call this function, since it doesn't return a promise.

Tip: you should probably have a squiggly under the `await` indicating that.